### PR TITLE
posts: add blog post about team communication

### PR DESCRIPTION
**Pull request by adrienjoly**
When done, this blog post will be published at https://adrienjoly.com/posts/how-not-communicate/ and shared with my followers on Twitter, LinkedIn and others.

Click the "Files changes" tab above to see the content. From there, click the "Display the rich diff" button if you want to see how it will look like, and/or feel free to make suggestions as comments on the corresponding lines of the source.

Also, here's a PDF export of the blog post: [How not to communicate with your team _ Adrien Joly.pdf](https://github.com/adrienjoly/adrienjoly.github.com/files/4165041/How.not.to.communicate.with.your.team._.Adrien.Joly.pdf) (to see the illustrations in context)

Constructive feedback is welcome!

### CODE REVIEW

File: posts/how-not-communicate/index.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    width: 600px;
+    margin: 32px auto;
+  }
+</style>
+
+# How *Not* to Communicate With Your Team
+
+Most of my career (so far), I have lead teams as a "[benevolent dictator](https://producingoss.com/en/benevolent-dictator.html)". Meaning that I have the vision of what we want to achieve as a team, but I am also the one prescribing the way we implement that vision. As a consequence, I expect my team to follow my lead, and to object just if I overlook anything important. Implementing this leadership style has been very effective, until I joined Algolia.
+
+Besides the fact that, at Algolia, my team and I had more years of experience than my previous teams, the culture of the company made it ineffective to keep implementing this leadership style. So, with help from my manager, I had to take a step back and develop my soft skills in order to lead effectively in that new organization.
```
Comment (raphi) [inline posts/how-not-communicate/index.md]: You can link my Linkedin account on "my manager" if you want to 😛 https://www.linkedin.com/in/daguenet/

File: posts/how-not-communicate/index.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+In this article, I share three stories which show an objective that I failed to achieve with my team, and how I finally tackle it. Then, I present a few strategies that helped me better a more effective Tech Lead in that team.
+
+My goal is to help you communicate effectively with your team and avoid making the same mistakes as me. As a Tech Lead, or as a regular Individual Contributor.
+
+## *The Elephant in the Room*
+
+Two years ago, I joined Matthieu in the Crawler team. Matthieu had created the Crawler from scratch, 6 months prior. I had set a mission to myself: make the Crawler grow from a proof of concept into a product which our enterprise customers can confidently rely on.
```
Comment (raphi) [inline posts/how-not-communicate/index.md]: Could be cool to link the Crawler's page as well ;) shameless plug

File: posts/how-not-communicate/index.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+This advice worked better than I expected. First, we sat down to share our respective intentions, regrets and apologies. It was hard, but it helped us re-create trust and care between us. And after 3 discussions, I understood the vision that led Matthieu to create that feature, I agreed with it, and we finally converged to a decision that was working and motivating for both of us!
+
+![learning 1](./assets/learning1.jpg)
+
+What did I learn from this story? To move forward in the same direction, we must start by taking the time to *really* listen to each other. By understanding each other's vision, motivations and preferences, we can converge to solutions that will be implemented with care.
+
+## *Let's All Pair!*
+
+The following quarter, I wanted to help my team-mates grow. One was intimidated by the maintenance of our Kubernetes clusters. Another needed help to define an action plan for the development of a big feature. And another was stuck on how to test a feature he had implemented. So I set up pair programming sessions with each of them.
```
Comment (raphi) [inline posts/how-not-communicate/index.md]: Didn't you wrote a blog post about it?

File: posts/how-not-communicate/index.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+![Nope!](./assets/nope.jpg)
+
+As I was presenting the problem to the team, one of my colleagues interrupted me because he didn't agree with one of the points I was making. This interruption caught me off-guard. I lost the self-confidence I needed to properly deliver my message. So I was frustrated and a little bit mad at my colleague for not having let me finish.
+
+By chatting with my colleague after the meeting, I understood that the interruption was triggered by a misunderstanding that I could have avoided if I had been able to finish my intervention. Also, my colleague did not mean harm, he just wanted to save us some time. I could have anticipated his reaction. And therefore, I should have better prepared for it.
+
+![learning 3](./assets/learning3.jpg)
+
+Later, I learned a communication trick that would have avoided that situation: share my intent and expectations at the beginning of the meeting. By doing so, I could have defused the interruption by saying: “we agreed that I would welcome your feedback AFTER I'm done introducing the problem”, and then keep going.
```
Comment (raphi) [inline posts/how-not-communicate/index.md]: Maybe "tip" instead of "trick"? Looks less manipulative?

File: posts/how-not-communicate/index.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+- [Soft Skills Engineering](https://softskills.audio/), a weekly podcast that helped me cope with difficult situations, in a fun way.
+
+Also, don't hesitate to share your learnings and/or ask questions [on Twitter](https://twitter.com/intent/tweet?text=@adrienjoly%20My%20question%20in%20response%20to&url={{ site.url | url_encode }}{{ page.url | url_encode }}), or [on GitHub](https://github.com/adrienjoly/adrienjoly.github.com/issues/7).
+
+> I would like to thank:
+>
+> - [Raphaël Daguenet](https://www.linkedin.com/in/daguenet/) for supporting and helping me grow in my Tech Lead role, and for co-writing this post with me;
+> - AlexC for having told me about Personal README, and sharing some tips;
+> - PeterV for helping me improve the quality of this talk, before it was a blog post;
+> - And the Crawler team for their patience and understanding while I was figuring out my communication with them. Also, for having shared their Personal README and having accepted that I share these stories.
```
Comment (raphi) [inline posts/how-not-communicate/index.md]: Last LGTM 😢 👏 🚀 👍

File: posts/how-not-communicate/index.md (hunk 3 above)
Comment (adrienjoly) [reply posts/how-not-communicate/index.md]: Yes. If I understand well, you're suggesting to add a link to it, here?

Comment (raphi) [reply posts/how-not-communicate/index.md]: Yes if you think it makes sense, it's always good to link to other materials

**Pull request merged**